package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "SDK",
+  "devDependencies": {
+    "markdownlint": "^0.27.0",
+    "markdownlint-cli": "^0.33.0",
+    "prettier": "^2.8.3"
+  },
+  "scripts": {
+    "format:markdown": "prettier --write \"**/**/*.md\" --prose-wrap always --parser markdown",
```

Comment:
Już mnie to męczyło że nie ma formatowania

Reply:
A nie da rady tego samego osiągnąć przez wpis w editorconfig? :/ bo teraz dla samego formatowania MD doszła w tym repo zależność na node/yarn.

Plus nie bardzo rozumiem jakie są zalety wrapowania MD. Przy dowolnej niezerowej zmianie, leci diff na cały akapit/wszystkie linijki często. Już chyba każdy edytor ma możliwość auto-wrapowania linii w widoku.

W sensie, nawet github w PRach Ci zawija (patrz czerwona strona): 
![image](https://github.com/allegro/dotnet-sdk/assets/2671396/8397249a-a247-4284-807c-567e5cf58972)